### DISCUSSION TITLE
Pros and cons of Splink vs. AWS Entity Resolution

### DISCUSSION THREAD

**Discussion by seperman** [moj-analytical-services/splink#2026] [Q&A]
AWS has the entity resolution product:
https://aws.amazon.com/entity-resolution/

What are the pros and cons of Splink vs. going with the AWS entity resolution?

**Comment by aalexandersson** [comment]
I can only make a few basic comments. Some pros of Splink:

- Relatively simple to use by default with DuckDB. 
- Relatively easy to implement data use agreements with an on-premise backend such as DuckDB. 
- Popular free and open source software (FOSS).
 
Splink with the Amazon Athena backend would be the most direct comparison for "Splink" vs AWS Entity Resolution. A high-level diagram with both Amazon Athena and AWS Entity Resolution in it is at [https://github.com/aws-samples/patient-matching-with-aws-healthlake-and-aws-entity-resolution](https://github.com/aws-samples/patient-matching-with-aws-healthlake-and-aws-entity-resolution).

**Comment by seperman** [comment]
Ok, thanks for answering @aalexandersson 
After posting the question, I went and read a bit more about AWS entity resolution. It seemed more of a framework to use different backends for entity resolution. For example, you could use AWS entity resolution with the Experian, or Senzing, etc. backend.
